.github/workflows/build.yml

```diff
@@ -12,6 +12,7 @@ env:
 jobs:
   build:
     strategy:
+      fail-fast: false
```

Comment:
If we need this, then please move it to a separate PR. In general, I've been okay happy with failing PRs on the first failure since it normally indicates a real error that the PR author should fix.

Reply:
Of course, this was only intended for this branch, thanks for the reminder.